spanner/cloudexecutor/executor/cloud_client_executor_mapping.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package executor
+
+// cloud_client_executor_mapping.go handles mapping from executor actions to client library structure.
```

Comment:
* We need not add `cloud_client` as prefix of all files given these files are already under `cloudexecutor/executor` folder. Should this be named `ExecutorActionsMapper`. If you agree it might be good to create a subdirectory `/utility/mapper` and add different mappers in there. 
* There is a similar comment below for adding `ErrorCodeMapper`